README.md

```diff
@@ -0,0 +1,19 @@
+# PokemonTCGApp
+
+Hello, my name is Marcos Cifuentes, and I'm currently a trainee at Globant in Mobile Studio.
```

Comment:
you needn't have put your personal presentation.

Reply:
done, thanks